src/components/Filter/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { Fragment } from 'react';
+import {
+  AdvancedFilterCategoryModel,
+  AdvancedFilter,
+  FilterOptionModel,
+  FilterType
+} from '@osu-cass/sb-components';
+import { createFilters, sanitizeParams, paramsFromFilter } from './FilterHelper';
+import { CSEFilterOptions, CSEFilterParams } from '../../models/filter';
+
+export interface FilterProps {
+  options: CSEFilterOptions;
+  params: CSEFilterParams;
+  onUpdate: (filter: CSEFilterParams) => void;
+}
+
+export interface CSEAdvancedFilterModels {
+  gradeFilter: AdvancedFilterCategoryModel;
+  subjectFilter: AdvancedFilterCategoryModel;
+  claimFilter: AdvancedFilterCategoryModel;
+  targetFilter: AdvancedFilterCategoryModel;
+}
+
+export const Filter: React.SFC<FilterProps> = ({ options, params, onUpdate }) => {
```

Comment:
Great job on this component. This code is very clean and readable. Great re-use of `sb-components`